server/datastore/mysql/migrations/tables/migration.go

```diff
@@ -1,7 +1,33 @@
 package tables
 
-import "github.com/fleetdm/goose"
+import (
+	"database/sql"
+
+	"github.com/fleetdm/goose"
+)
 
 var (
 	MigrationClient = goose.New("migration_status_tables", goose.MySqlDialect{})
 )
+
+func columnExists(tx *sql.Tx, table, column string) bool {
+	var count int
+	err := tx.QueryRow(
+		`
+SELECT
+    count(*)
+FROM
+    information_schema.columns
+WHERE
+    TABLE_SCHEMA = DATABASE()
```

Comment:
Great catch!

Reply:
Oh! This is the difference. Ok, missed it the first time around :)